assets/src/stories-editor/blocks/amp-story-page/edit.js

```diff
@@ -338,7 +338,7 @@ class PageEdit extends Component {
 								{ !! mediaId &&
 								<MediaUploadCheck>
 									<Button onClick={ () => setAttributes( { mediaUrl: undefined, mediaId: undefined, mediaType: undefined } ) } isLink isDestructive>
-										{ VIDEO_BACKGROUND_TYPE === mediaType ? __( 'Remove Video', 'amp' ) : __( 'Remove image', 'amp' ) }
+										{ _x( 'Remove', 'background media', 'amp' ) }
```

Comment:
Good idea to simplify this to 'Remove.' It's inside the 'Background Media' section, so it's clear enough what it's removing:

<img width="723" alt="remove-here" src="https://user-images.githubusercontent.com/4063887/60304039-c6be0300-98fd-11e9-9284-5b763de61c81.png">